### PR TITLE
New Disk Partitioning Page

### PR DESCRIPTION
**Pull request by mdhorn**
Fixes Issue: #26

Changes proposed in this pull request:
- Single page for partitioning the target installation media
- More user-friendly design based on UX study feedback
- Force the user to conform the existing requirements on a single disk for installation

### CODE REVIEW

**Comment by dorileo** [comment]
If I'm manually editing a disk after I select a partition and return to the partitioning main screen the previously selected disk should be select when returning.

**Comment by dorileo** [comment]
The confirm button should only be enabled when the partition table is valid.

File: themes/clr-installer.theme [hunk 1]
```diff
@@ -151,3 +156,10 @@ Main.MenuContentBack=blue
 
 Main.MenuContentActiveText=blue
 Main.MenuContentActiveBack=cyan
+// -- REMOVE THIS with the disk_manual.go
```
Comment (dorileo) [inline themes/clr-installer.theme]: suppose you should remove this then?

File: storage/storage.go [hunk 2]
```diff
@@ -28,6 +28,7 @@ type BlockDevice struct {
 	MajorMinor      string           // major:minor device number
 	FsType          string           // filesystem type
 	UUID            string           // filesystem uuid
+	Serial          string           // device serial number
```
Comment (dorileo) [inline storage/storage.go]: I'm wondering why you needed Serial

Comment (dorileo) [reply storage/storage.go]: answered myself, to compare blk devices.

File: themes/clr-installer.theme (hunk 1 above)
Comment (mdhorn) [reply themes/clr-installer.theme]: done.

**Comment by mdhorn** [comment]
@dorileo Resolved the code review issues. Fixed the size warning issue. Please re-review.

**Comment by mdhorn** [comment]
@dorileo Please re-review
Focus issues resolved.
Added test coverage to avoid regression

**Pull request merged**